src/content/docs/en/guides/images.mdx

```diff
@@ -4,159 +4,702 @@ description: Learn how to use images in Astro.
+
+However, both of these properties are required for remote images and images stored in your `public/` folder as Astro is unable to analyze these files.
+
+##### format 
+
+You can optionally state the [image file type](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#common_image_file_types) output to be used.
+
+By default, the `<Image />` component will produce a `.webp` file.
+
+##### quality
+
+`quality` is an optional property that can either be:
+- a preset (`low`, `mid`, `high`, `max`) that is automatically normalized between formats.
+- a number from `0` to `100` (interpreted differently between formats).
+
+
+##### Additional properties
+
+In addition to the properties above, the `<Image />` component accepts all properties accepted by the HTML `<img>` tag.
+
+For example, you can provide a `class` to the final `<img>` element.
 
 ```astro title="src/pages/index.astro"
 ---
-import rocket from '../images/rocket.svg';
+import { Image } from 'astro:assets';
+import myImage from "../assets/my_image.png";
 ---
-<!-- Remote image on another server -->
-<img src="https://docs.astro.build/assets/logomark-light.png" width="25" alt="Astro">
 
-<!-- Local image stored at public/assets/stars.png -->
-<img src="/assets/stars.png" alt="A starry night sky.">
+<!-- `alt` is mandatory on the Image component -->
+<Image src={myImage} alt="" class="my-class" />
+```
+
+```astro
+<!-- Output -->
+<img
+  src="/_astro/my_image.hash.webp"
+  width="1600"
+  height="900"
+  decoding="async"
+  loading="lazy"
+  class="my-class"
+  alt=""
+/>
+```
+
+#### Setting Default Values
+
+Currently, there is no way to specify default values for all `<Image />` components. Required attributes should be set on each individual component.
+
+As an alternative, you can wrap these components in another Astro component for reuse. For example, you could create a component for your blog post images:
+
+```astro title="src/components/BlogPostImage.astro"
+---
+import { Image } from 'astro:assets';
+
+const {src, ...attrs} = Astro.props;
+---
+<Image src={src} {...attrs} />
+
+<style>
+  img :global(img), svg {
+    margin-block: 2.5rem;
+    border-radius: 0.75rem;
+  }
+</style>
+```
+
+### `<img>`
+
+The [Astro template syntax](/en/core-concepts/astro-syntax/) also supports writing an `<img>` tag directly, with full control over its final output. These images will not be processed and optimized.
+
+It accepts all HTML `<img>` tag properties, and the only required property is `src`.
+
+#### Local images in `src/`
+
+Local images must be **imported from the relative path** from the existing `.astro` file, or configure and use an [import alias](/en/guides/aliases/). Then, you can access the image's `src` and other properties to use in the `<img>` tag.
+
+For example, use the image's own `height` and `width` properties by default to avoid CLS and improve Core Web Vitals.
+
+```astro "myDog.src"
```

Comment:
Since this is an example of using `height` and `width`, I think these should be highlighted here.

```suggestion
```astro "myDog.width" "myDog.height"
```